theme/tests/test_user_profile.py

```diff
@@ -0,0 +1,10 @@
+from django.test import TestCase

```

Comment:
There were no tests around UserProfile so I added this just to ensure email_opt_out is false by default

Reply:
I did attempt to update the selenium tests to include this functionality but was not able to get selenium tests to run.  I checked the builds in jenkins, I don't see the tests running there either.

Reply:
Let's make a ticket for fixing selenium tasks in GH

Reply:
https://github.com/hydroshare/hydroshare/issues/2652